KubeArmor/core/crioHandler.go

```diff
@@ -266,6 +291,20 @@ func (dm *KubeArmorDaemon) UpdateCrioContainer(ctx context.Context, containerID,
 			dm.ContainersLock.Unlock()
 			return false
 		}
+		if !dm.K8sEnabled {
+			dm.EndPointsLock.Lock()
+			for _, ep := range dm.EndPoints {
+				if ep.EndPointName == dm.Containers[containerID].ContainerName {
+					for i, c := range ep.Containers {
+						if c == containerID {
+							ep.Containers = append(ep.Containers[:i], ep.Containers[i+1:]...)
+							break
+						}
+					}
+				}
```

Comment:
It is usually preferred to reduce the nestings.
```suggestion
				if ep.EndPointName != dm.Containers[containerID].ContainerName {
				    continue
				}
				for i, c := range ep.Containers {
					if c != containerID {
					   continue
					}
					ep.Containers = append(ep.Containers[:i], ep.Containers[i+1:]...)
					break
				}
```